### PR TITLE
session storage

### LINKED ISSUE #19: Persistent sessions on server side

### PR DESCRIPTION
**Pull request by jlurgo**
- added file session storage to keep users logged in after server restarts/updates
- excluded session and accounts files from nodemon file monitor to avoid restarts on user or sessions updates

closes #19

### CODE REVIEW

File: src/server/main.ts [hunk 1]
```diff
@@ -17,9 +20,15 @@ const port = process.env.PORT || 3000;
 
 const app = express();
 app.use(express.json());
+FileStore(session);
+const fileStoreOptions = {
+  path: path.dirname(new URL(import.meta.url).pathname) + '/sessions',
```
Comment (chfritz) [inline src/server/main.ts]: Shouldn't these run-time files live outside of the `src/` tree? Maybe we can just use `${process.env.PWD}/var` and/or let the user specify a folder in their .env file, similar to https://github.com/transitiverobotics/transitive/blob/main/cloud/sample.env#L9?

Comment (jlurgo) [reply src/server/main.ts]: in that case I can also drop the nodemon handling

Comment (jlurgo) [reply src/server/main.ts]: Done!

**Review by chfritz — APPROVED** [review]
Nice!

**Pull request merged**